pull-request/src/app.js

```diff
@@ -7,7 +7,9 @@ var Person = function(first, last, middle) {
 Person.prototype = {
 
     whoAreYou : function() {
-        return this.first + (this.middle ? ' ' + this.middle: '') + ' ' + this.last;
+        // return this.first + (this.middle ? ' ' + this.middle: '') + ' ' + this.last;
+        // FIXME: Swap first and last name
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Take the required action to fix the issue indicated by this comment. ([javascript:S1134](http://localhost:9730/coding_rules#rule_key=javascript%3AS1134))

[See it in SonarQube](http://localhost:9730/project/issues?id=training%3Apull-request&pullRequest=17&issues=AWa_Tc5p9tEDrgOR9l3h&open=AWa_Tc5p9tEDrgOR9l3h)

[ISSUE_KEY]: AWa_Tc5p9tEDrgOR9l3h